config/policy-controller/kustomization.yaml

```diff
@@ -1,5 +1,5 @@
 resources:
-- github.com/Kuadrant/multicluster-gateway-controller/config/policy-controller/default?ref=main
+- github.com/Kuadrant/multicluster-gateway-controller/config/policy-controller/default?ref=release-0.2
```

Comment:
I'd say that these kind of version pointing in our manifests in main branch, should point to floating `latest` tag or `main` branch, and use the specific version for release branches. Does it make sense?

Reply:
on a second thought, if we want to treat the policy controller as an external dependencies, say, like istio, it's a different story

Reply:
issue with that is, any time something changes in the policy controller repo yamls, it will mean the verify-bundle will fail in this repo

Reply:
Authorino is listed as a dependency and it points it to main: https://github.com/Kuadrant/kuadrant-operator/blob/61711f31372d2e72c28d5c0dceaa92ded8a00983/config/dependencies/authorino/kustomization.yaml#L2

Reply:
In fact, I think the policy controller could be treated the same way. Why not having it under `config/dependencies`?

Reply:
so if something in authorino CRD changes then the bundle here also has to be regenerated? If that is the behaviour we would prefer I am ok with it, it had just seemed a bit of a pain rather than pinning to latest release

Reply:
This really depends what you are trying to achieve. If you want the TLSPolicy and DNSPolicy CRDs and policy controller deployment included in the kuadrant-operator bundle, then adding them to config/dependencies probably won't work. AFAIK, none of those dependencies are pulled into the bundle and the likes of `kuadrant-operator/config/dependencies/authorino/kustomization.yaml` is only used where you want to run everything without OLM. In a normal OLM installation authorinio-operator is installed as an OLM dependency since it has it's own bundle, policy-controller doesn't.

Reply:
Makes sense. Indeed the APIs defined by those dependencies are not part of Kuadrant Operator's bundle.

Reply:
Ok so only question is whether it should ref main or not. I am thinking not at least initially

Reply:
About fixing a version or using latest, I think the logic behind it was that:
- components we control we default to latest – and therefore we pick up changes in those that affect Kuadrant Operator as soon as possible, whereas
- components we do not control we fix a version – and therefore ensure stability of Kuadrant Operator before externalities going on on those projects.

Reply:
Ok so then we don't change this one and we close this PR?